staging/src/k8s.io/client-go/examples/third-party-resources/apis/tpr/v1/register.go

```diff
@@ -24,28 +24,25 @@ import (
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
 )
 
-// GroupName is the group name use in this package
+// AddToScheme applies all the functions stored in the SchemeBuilder value to a scheme.
+func AddToScheme(s *runtime.Scheme) error {
```

Comment:
Per [your suggestion](https://github.com/kubernetes/kubernetes/pull/45910/files#r116879919), @caesarxuchao, I removed commit fa8b8b9b5f557d63ecaec8911e680548bf3087a8 from this proposal. PTAL.